lxd/api_cluster.go

```diff
@@ -2749,7 +2749,7 @@ func evacuateClusterMember(d *Daemon, r *http.Request, mode string) response.Res
 				}
 
 				// Mark the instance as RUNNING in volatile so its state can be properly restored.
-				inst.VolatileSet(map[string]string{"volatile.last_state.power": "RUNNING"})
+				_ = inst.VolatileSet(map[string]string{"volatile.last_state.power": "RUNNING"})
```

Comment:
Probably should log this error as a warning as if this fails the instance won't be properly restored.